src/components/dao/dao.js

```diff
@@ -60,6 +59,40 @@ export default class Dao extends Component {
 
     }
 
+    proposalAlreadyExists() {
+
+        const { addr } = this.state;
+        const multicall = `${this.state.addr.value}.${window.nearConfig.MULTICALL_FACTORY_ADDRESS}`;
+        const sputnik = `${addr.value}.${window.nearConfig.SPUTNIK_V2_FACTORY_ADDRESS}`;
+
+        return view(
+            sputnik,
+            "get_last_proposal_id",
+            {}
+        )
+            .then(res => view(
+                sputnik,
+                "get_proposals",
+                {
+                    from_index: res < 100 ? 0 : res - 100,
+                    limit: 100
+                }
+            ))
+            .then(res => {
+
+                const proposals = res.filter(p => 
+                    p.description === `create multicall instance for this DAO at ${multicall}` &&
```

Comment:
It's better to check if proposal is function call, if target is multicall factory and if function name is 'create' to determine if the proposal id for making a multicall instance for the DAO.